app/css/openmrs-addonmanager.css

```diff
@@ -83,6 +83,21 @@ header>ul, a {
     color: #eeeeee;
     padding-right: 2px;
     margin-top: 15px;
+    font-size: 16px;
+}
+
+#curUser {
```

Comment:
We could rename this to better define what it does or what it is for and use the naming convention we currently follow. Something like 'account-styling'

Reply:
Yeah, makes sense